### PR TITLE
🌐 [i18n-KO] Translated `bertweet.md` to Korean

### PR DESCRIPTION
**Pull request by ahnjj**
# What does this PR do?

Translated the `bertweet.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [ ] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [ ] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

@chhaewxn, @ahnjj, @jun048098, @fabxoe

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

 @stevhliu May you please review this PR?

### CODE REVIEW

File: docs/source/ko/model_doc/bertweet.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# BERTweet [[bertweet]]
+
+## 개요 [[overview]]
+
+BERTweet 모델은 Dat Quoc Nguyen, Thanh Vu, Anh Tuan Nguyen에 의해 [BERTweet: 영어 트윗을 위해 사전 학습된 언어 모델](https://www.aclweb.org/anthology/2020.emnlp-demos.2.pdf) 에서 제안되었습니다.
```
[the remark this answers is not in the document]
Comment (jun048098) [reply docs/source/ko/model_doc/bertweet.md:21]: 논문명이라 영문 추천합니다

**Review by jun048098 — COMMENTED** [review]
fix: docs suggetion

File: docs/source/ko/model_doc/bertweet.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+BERTweet 모델은 Dat Quoc Nguyen, Thanh Vu, Anh Tuan Nguyen에 의해 [BERTweet: A pre-trained language model for English Tweets](https://www.aclweb.org/anthology/2020.emnlp-demos.2.pdf) 에서 제안되었습니다.
+
+해당 논문의 초록 :
+
+*영어 트윗을 위한 최초의 공개 대규모 사전 학습된 언어 모델인 BERTweet을 소개합니다. 
+BERTweet은 BERT-base(Devlin et al., 2019)와 동일한 아키텍처를 가지고 있으며, RoBERTa 사전 학습 절차(Liu et al., 2019)를 사용하여 학습되었습니다. 
+실험 결과, BERTweet은 강력한 기준 모델인 RoBERTa-base 및 XLM-R-base(Conneau et al., 2020)의 성능을 능가하여 세 가지 트윗 NLP 작업(품사 태깅, 개체명 인식, 텍스트 분류)에서 이전 최신 모델보다 더 나은 성능을 보여주었습니다.*
+
+이 모델은 [dqnguyen](https://huggingface.co/dqnguyen) 께서 기여하셨습니다. 원본 코드는 [여기](https://github.com/VinAIResearch/BERTweet).에서 확인할 수 있습니다.
```
Comment (fabxoe) [inline docs/source/ko/model_doc/bertweet.md:29]:
```suggestion
이 모델은 [dqnguyen](https://huggingface.co/dqnguyen)님이 기여했습니다. 원본 코드는 [여기](https://github.com/VinAIResearch/BERTweet)에서 확인할 수 있습니다.
```
제가 알기로는 2중 높임은 문법적으로 사용하지 않아야 합니다.("~님이 ~했습니다". "~가 ~하셨다.")

**Review by stevhliu — APPROVED** [review]
LGTM!

**Pull request merged**